src/org/ssgwt/client/validation/validators/StringRegexValidator.java

```diff
@@ -78,17 +78,22 @@
      * The regular expression patterns string to be used to validate an Telephopne numbers.
      */
     public static final String REGEX_TELEPHONE_NUMBER_PATTERN = "(^[\\+][0-9]{0,15}$)|(^0{1}[0-9]{0,14}$)";
-    
+
+    /**
+     * The regular expression patterns string to be used to validate decimal numbers.
+     */
+    public static final String REGEX_FLOATINGPOINT_NUMBER_PATTERN = "(^[+-]?([0-9]*[.])?[0-9]+$)";
```

Comment:
is this being used?

Reply:
yes, I have a validator that check for valid numbers